test/s3-benchmarks/src/main/java/software/amazon/awssdk/s3benchmarks/TransferManagerBenchmarkConfig.java

```diff
@@ -26,6 +26,7 @@ public final class TransferManagerBenchmarkConfig {
     private final ChecksumAlgorithm checksumAlgorithm;
     private final Integer iteration;
     private final Long contentLengthInMb;
+    private final Long timeout;
```

Comment:
Should we use Duration so that we don't need to worry about unit?

Reply:
done